docs/snippets/gradual-rollout-intro.md

```diff
@@ -0,0 +1,13 @@
+# Configuring gradual roll-out of traffic to Revisions
+
+If your traffic configuration points to a Configuration target instead of a Revision target, when a new Revision is created and ready, 100% of the traffic from the target is immediately shifted to the new Revision.
+
+This might cause requests to become backed up, either at the QP or Activator, and after a while the requests might expire or be rejected by the QP.
```

Comment:
```suggestion
This might make the request queue too long, either at the QP or Activator, and cause the requests to expire or be rejected by the QP.
```

Reply:
More subjective a suggestion, but I suspect ESL admins are much much more used to reading "become backed up" as "made a back-up copy" and might trip up.